### ISSUE TITLE
TreeConstructor.NodeContents.Tag.close is actually optional

### ISSUE THREAD

**Issue by BlackGlory** [mykolaharmash/hyntax#32]
README:
```ts
interface Tag {
  name: string
  selfClosing: boolean
  openStart: Tokenizer.Token<Tokenizer.TokenTypes.OpenTagStart>
  attributes?: TagAttribute[]
  openEnd: Tokenizer.Token<Tokenizer.TokenTypes.OpenTagEnd>
  children?: AnyNode[]
  close: Tokenizer.Token<Tokenizer.TokenTypes.CloseTag>
}
```

TypeScript declarations:
https://github.com/mykolaharmash/hyntax/blob/96bb1d2e5f56c98428f93c2ca13945fe6db9a227/index.d.ts#L222

Actual AST(<https://astexplorer.net/#/gist/a0e72b2be44f5c23214c4c238093510c/39df950f9fcd76b7df249427dbbca2f4809cf235>):
```js
{
  "nodeType": "document",
  "content": {
    "children": [
      {
        "nodeType": "tag",
        "parentRef": "[Circular ~]",
        "content": {
          "openStart": {
            "type": "token:open-tag-start",
            "content": "<br",
            "startPosition": 0,
            "endPosition": 2
          },
          "name": "br",
          "openEnd": {
            "type": "token:open-tag-end",
            "content": "/>",
            "startPosition": 4,
            "endPosition": 5
          },
          "selfClosing": true
        }
      }
    ]
  }
}
```

**Comment by mykolaharmash** [comment]
Hi! Thank you for pointing this out, you're right. Will fix.

**Comment by mykolaharmash** [comment]
Fixed, please update to `v1.1.9`
